cmd/acra-keys/keys/generate.go

```diff
@@ -298,7 +298,8 @@ func (g *GenerateKeySubcommand) Execute() {
 		}
 	}
 
-	keyLoader, err := keyloader.GetInitializedMasterKeyLoader(g.CommonKeyStoreParameters.KeyLoaderCLIOptions().KeystoreEncryptorType)
+	masterKeyLoaderFactory := keyloader.NewMasterKeyLoaderFactory(g.CommonKeyStoreParameters.KeyLoaderCLIOptions().KeystoreEncryptorType)
+	keyLoader, err := keyloader.GetInitializedMasterKeyLoader(masterKeyLoaderFactory)
```

Comment:
This pattern repeats a couple of times. Are there any reasons not to move it into a separate function? Like `keyloader.{Build, Create, New, Assemble}MasterKeyLoader(loadStrategy string)`?